### DISCUSSION TITLE
Crazyflie cannot take off

### DISCUSSION THREAD

**Discussion by HaomiaoLuoLlanio** [bitcraze/discussions#350] [Q&A]
Hi!
Information:
Motion capture system: OptiTrack, Motive
Crazyflie: 2.1
Joystick: Xbox USB connection
ROS: crazyswarm, Rviz
I brought new crazyflies. I have assembled them and flashed the firmware. The crazyflies can connect to chooser... But, when I use the XBox USB send the take-off command to crazyflie, the crazyflie would drift in oneside,  some quadrotors flips over on the ground and the height they take off is so low like attaching to the ground. We think the problem is configuration of the Xbox joystick or the setup of thrust but we don't know how to solve that. BTW, we already test the motors, all 4 motors can rotate.
Hope you can reply and figure out the problem. Thank you sincerely!

**Comment by knmcguire** [comment]
Hi! 

Sorry for the delayed answer, we were in Japan for a conference.

Since this is a Crazyswarm(1) question, you'll need to ask that on the required discussion forum here: https://github.com/USC-ACTLab/crazyswarm/discussions. This will keep it easier for us to seperate the issues and therefore the maintainers are aware of the issue. 

Just a small tip: You'll need to make sure that the positioning of the crazyflie is correct, which is something you can check in RViz just by the hand.

For the rest, please ask the question on the [crazyswarm discussion forum](https://github.com/USC-ACTLab/crazyswarm/discussions), with the settings you set in the launch file as well.

**Reply by HaomiaoLuoLlanio** [reply]
Thank you for your reply. I am trying to ask then for help.
